aip_x1_launch/launch/pointcloud_preprocessor.launch.py

```diff
@@ -88,32 +88,33 @@ def launch_setup(context, *args, **kwargs):
 
     # set container to run all required components in the same process
     container = ComposableNodeContainer(
-        name="pointcloud_preprocessor_container",
-        namespace="pointcloud_preprocessor",
+        name=LaunchConfiguration("container_name"),
+        namespace="",
         package="rclcpp_components",
         executable=LaunchConfiguration("container_executable"),
         composable_node_descriptions=[],
+        condition=UnlessCondition(LaunchConfiguration("use_pointcloud_container")),
         output="screen",
     )
 
     # load concat or passthrough filter
     concat_loader = LoadComposableNodes(
         composable_node_descriptions=[concat_component],
-        target_container=container,
-        condition=launch.conditions.IfCondition(LaunchConfiguration("use_concat_filter")),
+        target_container=container
+        if UnlessCondition(LaunchConfiguration("use_pointcloud_container")).evaluate(context)
+        else LaunchConfiguration("container_name"),
+        condition=IfCondition(LaunchConfiguration("use_concat_filter")),
```

Comment:
please create variable same as the https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/147#discussion_r762712219

Reply:
fix in https://github.com/tier4/AutowareArchitectureProposal_aip_launcher/pull/8/commits/a2100e7a155e97514682e624a5f8aba00d01ad45